### PR TITLE
Rabbitmq support

### PR DESCRIPTION
**Pull request by jbadiapa**

### CODE REVIEW

File: roles/rabbitmq/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+sensu_rabbitmq_vhost: /sensu
+sensu_rabbitmq_user: sensu
+sensu_rabbitmq_password: sensu
+
```
Comment (larsks) [inline roles/rabbitmq/defaults/main.yml]: Shouldn't this configuration be provided by the sensu role?

File: roles/rabbitmq/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+- name: Create rabbitmq vhost
+  rabbitmq_vhost:
+   name: '{{ sensu_rabbitmq_vhost }}'
+   state: present
+  notify: Restart rabbitmq
+
+# Add user to server and assign full access control on / vhost.
+# The user doesn't have permission rules for other vhosts
+- name: Configure rabbitmq permissions
```
Comment (larsks) [inline roles/rabbitmq/tasks/main.yml]: Same comment here; the sensu-specific configuration should probably live in the sensu role.

File: roles/rabbitmq/defaults/main.yml (hunk 1 above)
Comment (jbadiapa) [reply roles/rabbitmq/defaults/main.yml]: You're right however this should run on the rabbitmq role will be installed.

**Comment by mrunge** [comment]
following the pattern to put every package name into vars, you might want to add that to this PR

**Comment by paramite** [comment]
+1

**Comment by paramite** [comment]
So does anybody has objections againts merging this?

**Review by mrunge — APPROVED** [review]
looks good to me

**Review by larsks — COMMENTED** [review]
I think we're going to need a service handler for restarting rabbitmq after configuration changes.

**Comment by jbadiapa** [comment]
Larsks, you're right.
As its not needed from the start I deleted.
BTW, Its added with the  sensu support. You can take a look at https://github.com/jbadiapa/opstools-ansible-1/tree/sensu

**Comment by paramite** [comment]
@jbadiapa it should be part of this patch. In case we will need to revert sensu patch for some reason in future, you won't loose functionality in rabbitmq role. As soon as you will add it to this patch I'm gonna merge.

**Pull request merged**